iceoryx_posh/test/moduletests/test_roudi_cmd_line_parser_config_file_option.cpp

```diff
@@ -97,10 +103,65 @@ TEST_F(CmdLineParserConfigFileOption_test, HelpLongOptionLeadsProgrammNotRunning
     args[0] = &appName[0];
     args[1] = &option[0];
 
-    iox::config::CmdLineParserConfigFileOption sut;
+    CmdLineParserConfigFileOption sut;
     auto result = sut.parse(numberOfArgs, args);
 
-    EXPECT_THAT(result.has_error(), Eq(false));
-    EXPECT_THAT(result.value().run, Eq(false));
+    EXPECT_FALSE(result.has_error());
+    EXPECT_FALSE(result.value().run);
 }
+
+TEST_F(CmdLineParserConfigFileOption_test, WrongOptionLeadsUnkownOptionResult)
+{
+    constexpr uint8_t numberOfArgs{2U};
```

Comment:
coding guideline states the all constexpr should be written in UPPER_SNAKE_CASE.

Reply:
Looks like it's done.

Reply:
Btw, it looks like the original code came from PR #523. So these finding was missed in the review of this PR!